### PR TITLE
translate: fix translation and grammar on vim's guide index.md

### PR DESCRIPTION
**Pull request by jeffersonataide**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] My article does not contain shortened URLs or affiliate links.

If your pull request closes a GitHub issue, replace the XXXXX below with the issue number.

Closes #XXXXX

### CODE REVIEW

**Review by CleoAguiar — CHANGES_REQUESTED** [review]

File: guide/portuguese/vim/index.md [hunk 1]
```diff
@@ -6,25 +6,26 @@ localeTitle: Vim
 
 [![logo](https://upload.wikimedia.org/wikipedia/commons/thumb/9/9f/Vimlogo.svg/75px-Vimlogo.svg.png "Logotipo Vim")](https://vim.sourceforge.io)
 
-O Vim (Vi IMproved) é um editor de texto altamente configurável, criado para tornar a criação e alteração de qualquer tipo de texto muito eficiente. Está incluído como "vi" na maioria dos sistemas UNIX, que também inclui o macOS da Apple. O Vim é uma grande melhoria em relação ao seu predecessor _vi_ , que foi desenvolvido há mais de 40 anos e ainda está sendo usado.
+O Vim (Vi IMproved) é um editor de texto altamente configurável, criado para tornar a criação e alteração de qualquer tipo de texto altamente eficiente. O mesmo é incluído como "vi" na maioria dos sistemas UNIX, o que inclui o macOS da Apple. O Vim é uma grande melhoria em relação ao seu predecessor *vi* , que foi desenvolvido há mais de 40 anos, e continua sendo usado.
 
-O Vim é estável ao rock e possui muitos recursos, incluindo:
+O Vim é extremamente estável e entre os seus muitos recursos estao inclusos:
```
[the remark this answers is not in the document]
Comment (RandellDawson) [reply guide/portuguese/vim/index.md]: @CleoAguiar Your suggested change is not different than what the user already has.  Make sure to corrected verbiage/code you expect to see in the `suggestion`.

Thank you.

[the remark this answers is not in the document]
Comment (CleoAguiar) [reply guide/portuguese/vim/index.md]: Sorry, I've updated the suggestion

[the remark this answers is not in the document]
Comment (RandellDawson) [reply guide/portuguese/vim/index.md]: I made the change on behalf of the user.

**Review by RandellDawson — APPROVED** [review]

**Comment by RandellDawson** [comment]
Thank you for your contribution to the page! 👍
We're happy to accept these changes, and look forward to future contributions. 🎉

**Pull request merged**